src/routes/console/project-[project]/storage/bucket-[bucket]/settings/+page.svelte

```diff
@@ -282,7 +282,7 @@
 
         <Form onSubmit={updateFileSecurity}>
             <CardGrid>
-                <Heading tag="h6" size="7">File Security</Heading>
+                <Heading tag="h6" size="7" id="file-security">File Security</Heading>
```

Comment:
@ArmanNik @TGlide Everything should be in sentence case

This should be `File security`

Do y'all mind going around to check this is true everywhere? Also ping design if they had title case somewhere.